data/fields/tee.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "tee",
+    "type": "combo",
```

Comment:
this should probably be a `semiCombo` field, as tees can occasionally have multiple "colors"